sites/docs/src/content/docs/tutorials/tests_and_test_data/components/03_project_setup.md

```diff
@@ -0,0 +1,392 @@
+
+    // Input data
+    input = "${projectDir}/assets/samplesheet.csv"
+
+    // Genome references
+    fasta       = "${params.pipelines_testdata_base_path}/reference/genome.fa.gz"
+    fasta_index = "${params.pipelines_testdata_base_path}/reference/genome.fa.fai"
+}
+
+process {
+    resourceLimits = [
+        cpus: 4,
+        memory: '15.GB',
+        time: '1.h'
+    ]
+}
+```
+
+## Test Data Structure
+
+### Real Samplesheet (assets/samplesheet.csv)
+
+```csv
+sample,fastq_1,fastq_2,genome
+SRR389222_sub1,https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz,,
+SRR389222_sub2,https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz,,
+SRR389222_sub3,https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub3.fastq.gz,,
+Ecoli_10K_methylated,https://github.com/nf-core/test-datasets/raw/methylseq/testdata/Ecoli_10K_methylated_R1.fastq.gz,https://github.com/nf-core/test-datasets/raw/methylseq/testdata/Ecoli_10K_methylated_R2.fastq.gz,
+```
+
+### Features of the Test Samplesheet
+
+1. **Mixed data types**: Single-end and paired-end samples
+2. **Remote URLs**: Direct links to GitHub test datasets
+3. **Minimal size**: Small datasets for fast testing
+4. **Real data**: Actual methylated sequencing data, not synthetic
+
+## Input Validation Schema
+
+### Schema Structure (assets/schema_input.json)
+
+```json
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/methylseq/master/assets/schema_input.json",
+    "title": "nf-core/methylseq pipeline - params.input schema",
+    "description": "Schema for the file provided with params.input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "sample": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["id"]
+            },
+            "fastq_1": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            },
+            "fastq_2": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            }
+        },
+        "required": ["sample", "fastq_1"]
+    }
+}
+```
+
+## Project Initialization Steps
+
+### 1. Initialize nf-test (if not present)
+
+```bash
+# Initialize nf-test in the project
```

Comment:
I'm a bit confused why we've jumped to this now? I think without some text I'm not following the 'storyline'